ChangeLog

```diff
@@ -64,6 +64,9 @@ Features
    * Add a new function mbedtls_asn1_write_named_bitstring() to write ASN.1
      named bitstring in DER as required by RFC 5280 Appendix B.
 
+Mbed Crypto
```

Comment:
Please add release note items to [our upcoming release's release notes page](https://confluence.arm.com/display/IoTBU/Mbed+Crypto+1.1.0+Release+Notes). We can edit them and get them into shape before publishing separately from code changes.

Reply:
I have added the text suggested by @gilles-peskine-arm to the page.